content/doc/developer/cli/writing-cli-commands.adoc

```diff
@@ -0,0 +1,65 @@
+---
+title: Writing CLI commannds
+layout: developersection
```

Comment:
Please note that the developer docs have two general kinds of page: sections, and guides. The latter is intended as a straightforward "how to", while the former is general documentation.

It looks like this page would make a better guide?